### PR TITLE
Rewrite website

### PR DESCRIPTION
**Pull request by kuzdogan**

### CODE REVIEW

File: data/blog/2024/1-what.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: 1) What
```
Comment (ligi) [inline data/blog/2024/1-what.mdx:2]:
```suggestion
title: What  to hack at ETHBerlin04
```

File: data/blog/2024/1-what.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+date: 2024-03-26T16:06:04.000Z
+lastmod: 2024-07-04T12:57:33.000Z
+tags: ['ETHBerlin', '04']
+authors: ['Franzi', 'Helena']
+images: ['/static/images/2024/banner-3.png']
+---
+
+## 2) To hack at ETHBerlin04
+
```
Comment (ligi) [inline data/blog/2024/1-what.mdx:11]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: data/blog/2024/1-what.mdx (hunk 1 above)
Comment (kuzdogan) [reply data/blog/2024/1-what.mdx:2]: I think this was intentional :) https://x.com/SBF_FTX/status/1591989554881658880

File: data/blog/2024/1-what.mdx (hunk 2 above)
Comment (kuzdogan) [reply data/blog/2024/1-what.mdx:11]: This is also building up onto the title above

**Pull request merged**